src/common/constants.ts

```diff
@@ -16,4 +17,10 @@ export const FILES_PATHS = {
 // *********************************
 
 export const tempDirectory = path.join(__dirname, '..', '/temp')
-export const svgDirectory = path.join(__dirname, '..', '/svg')
+const svgDirectory = path.join(__dirname, '..', '/svg')
+const outlinedDirectory = path.join(__dirname, '..', '/svg-outlined')
+
+export const ThemesDirectories: themesDictionary = {
+  [iconsTheme.svg]: svgDirectory,
+  [iconsTheme.outlined]: outlinedDirectory
+}
```

Comment:
I've created an enum `iconsTheme` where all themes can be set there, and then created an `themesDictionary` type specify that this dictionary must have the themes as a keys. 
then, in this dictionary, We set the directory corresponding to each theme. 

What do you think?

Reply:
looks good